ts-client/src/dlmm/index.ts

```diff
@@ -828,6 +829,133 @@ export class DLMM {
     return (min ? binId.floor() : binId.ceil()).toNumber();
   }
 
+  /**
+  * The function `getLbPairLockInfo` retrieves all pair positions that has locked liquidity.
+  * @returns The function `getLbPairLockInfo` returns a `Promise` that resolves to a `PairLockInfo`
+  * object. The `PairLockInfo` object contains an array of `PositionLockInfo` objects.
+  */
+  public async getLbPairLockInfo(): Promise<PairLockInfo> {
```

Comment:
we need to add param to indicate `minimumTime` for those positions
So we just return information of positions, that has `lockReleasePoint > currentTime + minimumTime`